New version: GaijinNetwork.StarConflict version 1.0.3.10

Dragon1573 opened the pull request:
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #217877

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.9 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.9.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/218010)

Dragon1573 committed:
New version: GaijinNetwork.StarConflict version 1.0.3.10

Signed-off-by: Dragon1573 <49941141+Dragon1573@users.noreply.github.com>

DuckDuckStudio commented on manifests/g/GaijinNetwork/StarConflict/1.0.3.10/GaijinNetwork.StarConflict.installer.yaml:
```diff
@@ -0,0 +1,25 @@
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: StarConflictLauncher_is1
```
```suggestion
```

我不认为 `软件包名_is1` 是一个正确的 `ProductCode`，这通常发生在作者没有在iss文件中指定 `ProductCode` 时（至少我遇到的）。
我遇到过的这种 `ProductCode` 的清单似乎都是由 Komac 创建的。

Dragon1573 committed:
Apply suggestions from code review

- Remove "Product Code" property, since it's not configured in ISS file.

Co-authored-by: 鸭鸭「カモ」 <Yzcbs123@163.com>

stephengillie commented:
> === manifests\g\GaijinNetwork\StarConflict\1.0.3.10 === Missing Properties value based on version 1.0.3.66: MinimumOSVersion

Hi @Dragon1573,

Is the `MinimumOSVersion` removal significant?

Dragon1573 commented on manifests/g/GaijinNetwork/StarConflict/1.0.3.10/GaijinNetwork.StarConflict.installer.yaml:
```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GaijinNetwork.StarConflict
+PackageVersion: 1.0.3.10
```
```suggestion
PackageVersion: 1.0.3.10
MinimumOSVersion: 10.0.0.0
```

Dragon1573 committed:
Apply suggestions from code review

- Re-add "Minimum OS Version" property according to previously exists manifest.

stephengillie commented:
Verify:
![image](https://github.com/user-attachments/assets/cce8c1ff-b757-41e2-aa91-7d687843329d)
![image](https://github.com/user-attachments/assets/717f94b0-90aa-41d1-bd4d-3693a56917a7)

DuckDuckStudio commented:
Maybe it's a version difference between the game and the launcher?
If so, should the package identifier be changed to GaijinNetwork.StarConflictLauncher (This was the first identifier of the package, and was changed to the current identifier in https://github.com/microsoft/winget-pkgs/pull/55914 .)?

Dragon1573 commented:
I think it's expected. The `InstallerUrl` names this file as `starconf_launcher_1.0.3.10.exe`. It should be an install wizard of the game launcher instead of the main game itself.

@stephengillie - As your screenshot, the main game itself is around 9.1GiB and should be downloaded by the launcher externally.
